plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -57,6 +57,8 @@ internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
 
   private val lifecycleListeners = CopyOnWriteArraySet<CameraAnimationsLifecycleListener>()
 
+  override var debugMode: Boolean = true
```

Comment:
Can we make it false by default? Users who want to see the log can enable it intentionally, but I don't think it's a common requirement for most users.

Reply:
I've set it to true in order to not make it a breaking change in terms of behaviour as now those logs are always enabled.

Reply:
I also would prefer false as default. I don't consider logs to be part of breaking changes.

Reply:
thanks for your thoughts here, folks, will disable by default 👍